build_tools/bazel/native_binary.bzl

```diff
@@ -52,6 +52,8 @@ def _native_binary_impl(ctx):
     return [default_info]
 
 def _native_test_impl(ctx):
+    if not ctx.attr.timeout:
+        ctx.attr.timeout = "short"
```

Comment:
I'm a bit leery of this approach because I think it makes this rule not conform to the spec for a test rule in Bazel. I know that functionally it should be the same, but I think I'd prefer if we kept the starlark rules the same and wrapped them in a macro that defaulted timeout to "short". The other reason to do this is that bazel_skylib has `native_test` and `native_binary` rules and we probably want to eventually adopt them. We didn't originally because they were broken and missing several features and no one was responding to my upstreamining PRs. Those did eventually get merged though. I think now the diff is just in support for the `env` attribute.